### ISSUE TITLE
Mutliple monitors to single monitor ⇒ Unexpected Error

### ISSUE THREAD

**Issue by jilleJr** [RoosterDragon/Desktop-Ponies#10]
Switched from 2 monitors to 1 and got this little error message when opening the options:

```
Unexpected Error - Desktop Ponies v1.58
An unexpected error occurred and Desktop Ponies must close. Please report this error so it can be fixed.
System.ArgumentOutOfRangeException: Värdet på 3600 är inte giltigt för Value. Value ska vara mellan 'Minimum' och 'Maximum'.
Parameternamn: Value
   vid System.Windows.Forms.NumericUpDown.set_Value(Decimal value)
   vid DesktopPonies.OptionsForm.RefreshOptions() i _______________________________________________Desktop Ponies\Desktop Ponies\DesktopPonies\OptionsForm.vb:rad 94
   vid DesktopPonies.OptionsForm.LoadInternal() i _______________________________________________Desktop Ponies\Desktop Ponies\DesktopPonies\OptionsForm.vb:rad 47
   vid System.Windows.Forms.Control.InvokeMarshaledCallbackDo(ThreadMethodEntry tme)
   vid System.Windows.Forms.Control.InvokeMarshaledCallbackHelper(Object obj)
   vid System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   vid System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   vid System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   vid System.Windows.Forms.Control.InvokeMarshaledCallback(ThreadMethodEntry tme)
   vid System.Windows.Forms.Control.InvokeMarshaledCallbacks()
2016-01-09 19:33:51Z
```

**Comment by RoosterDragon** [comment]
Did you switch from 2 to 1 whilst the program was running, or did you reopen the program after switching to 1?

**Comment by jilleJr** [comment]
Yes good point. I first had 2 monitors, set some custom settings on it so the pony area avoids the top and bottom by about 5-10%. Had the settings saved on my profile, and then later I removed the 2nd monitor (whilst the computer was shutdown), and after that the issue I mentioned appeared.

I can try to switch to a fresh profile. Lemme do that and come back later with the results

**Comment by jilleJr** [comment]
Yea so just switching to the `default` profile and then opening the settings worked, so I bet it was something with my profile that confused it.

Works now that I've recreated the profile :+1:

**Comment by RoosterDragon** [comment]
Fixed by 4083c8635ec340060cb251242ebfd7b51ad2e298
